hack/kcp-edge.sh

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env bash
+# Copyright 2023 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: $0 (start or stop | --user dev or kit | -v
+
+# Purpose: deploy the kcp-edge platform. The following components are created"
+#           (a) 1 kcp workspace: edge service provider workspace (espw)
+#           (b) 3 kcp-edge controllers: edge-scheduler, mailbox-controller and placement-translator
+
+# Assumption: kcp server is running.
```

Comment:
There are more assumptions.  Maybe better called preconditions.  One is checked below, regarding CRI.

Another is that `$KUBECONFIG` and `~/.kube/config` conspire to supply a collection of contexts that works for the commands in here.